hbase-hbtop/src/main/java/org/apache/hadoop/hbase/hbtop/HBTop.java

```diff
@@ -119,6 +120,7 @@ public int run(String[] args) throws Exception {
         try {
           delay = Integer.parseInt(commandLine.getOptionValue("delay"));
         } catch (NumberFormatException ignored) {
+          delay = 0;
```

Comment:
What is the intention here to assign 0 again?

Reply:
Just to keep the block from being empty. A comment works here too if it is less of an eyesore. I will try it.